docs/text-area.md

```diff
@@ -0,0 +1,204 @@
+# Textarea
+
+![simulator_screenshot_93D9FFAA-AC38-4CA6-94F0-E7A861FF5187](https://user-images.githubusercontent.com/20360870/202174447-930c4d36-5f12-448e-bd19-44f9d75d19b8.png)
+
+
+Adapt UI provides one theme for text area component with three styles and eight interaction states. Use this component to receive information from the user.
+
+#### Simple Usage
+
+```js
+import { TextArea } from "@adaptui/react-native-tailwind"
+export default function App() {
+  return (
+      <TextArea placeholder="Type Something...." />
+  )
+}
+```
+
+## Table of Contents
+
+  - [Themes](#themes)
+  - [Variant](#variant)
+  - [Size](#size)
+  - [Icon](#icon)
+  - [Interactions](#interactions)
+  - [Suffix](#suffix)
+  - [Props](#props)
+## Themes
+
+Adapt UI provides only one `base` theme for TextArea.
+
+![simulator_screenshot_CF05548F-87A9-4B9B-8298-42BCC812F415](https://user-images.githubusercontent.com/20360870/202165894-10ad9c82-ce7e-4251-be58-deac6ba98eb2.png)
+
+<details>
+
+<summary>
+  <b>Usage</b>
+</summary>
+
+### Usage
+
+```js
+import { TextArea, Slot, Icon } from "@adaptui/react-native-tailwind"
+export default function App() {
+  return (
+      <TextArea
+        placeholder="Type Something..."
+      />
+  )
+}
+
+```
+
+</details>
+
+## Variant
+
+Adapt UI provides four different input field components styles: `outline`,
+`subtle` and `underline`.
+
+You can use these various styled input field components based on the necessity
+and style you wish to provide to your design.
+
+![simulator_screenshot_93D9FFAA-AC38-4CA6-94F0-E7A861FF5187](https://user-images.githubusercontent.com/20360870/202174447-930c4d36-5f12-448e-bd19-44f9d75d19b8.png)
```

Comment:
The order above mentioned above and the preview do not match.

Reply:
fixed it